### PR TITLE
OGD-99: Send DCS Audit Events to SQS

### PR DESCRIPTION
**Pull request by dbes-gds**
- Resurrected the AWS SQS Client in Event Emitter.
- A new EventEmitterSQS client is available that saves audit events to an SQS Queue.
- New non-mandatory configuration property for the queue name added to configuration.
- Tests no longer run in Docker as none of the tests have any Python dependencies, which was the original motivation for running the tests under Docker.
- There are LocalStack integration tests for the SQS client which run in the LocalStack Docker container.
- Integration tests now run in a separate Gradle task.
- Travis configuration simplified following removal of Docker.

### CODE REVIEW

**Comment by dbes-gds** [comment]
The idea was to add the SQS functionality without forcing changes to the existing API Gateway code, making use of shared infrastructure.  Would be good to refactor and rename the API Gateway code once SQS has been delivered.  Prefer Option 2 to do this.

File: src/test/java/uk/gov/ida/eventemitter/utils/TestEvent.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        this.timestamp = timestamp;
+    }
+
+    @Override
+    public String getJSONString () {
+        final StringBuffer sb = new StringBuffer("TestAuditEvent{");
+        sb.append("eventId=").append(eventId);
+        sb.append(", timestamp=").append(timestamp);
+        sb.append('}');
+        return sb.toString();
```
Comment (bjgill) [inline src/test/java/uk/gov/ida/eventemitter/utils/TestEvent.java]: This doesn't look like JSON to me. Is this intentional?

**Comment by brenetic** [comment]
We do have the intention of removing the API Gateway in the Event System and putting events directly onto an SQS queue as this PR is intending for DCS. In which case it might be best to leave the refactoring until that happens as we might be able to remove a lot of code at the same time.

File: src/test/java/uk/gov/ida/eventemitter/utils/TestEvent.java (hunk 1 above)
Comment (dbes-gds) [reply src/test/java/uk/gov/ida/eventemitter/utils/TestEvent.java]: Have now delegated json serialisation to an ObjectMapper

**Pull request merged**